tests/integration/test_clip_text.py

```diff
@@ -7,7 +7,12 @@
 import torch
 from PIL import Image
 from jina import Flow, Document, DocumentArray, requests, Executor
-from jinahub.encoder.ClipTextEncoder import ClipTextEncoder
+#from jinahub.encoder.ClipTextEncoder import ClipTextEncoder
+import sys
+
+sys.path.insert(1, '../..')
```

Comment:
what is this?

Reply:
so that ClipTextEncoder can be imported from module in the root directory

Reply:
No, this is nnot valid, no other PR is doing this